api/lib/opentelemetry/context.rb

```diff
@@ -12,6 +12,10 @@ module OpenTelemetry
   class Context
     KEY = :__opentelemetry_context__
     EMPTY_ENTRIES = {}.freeze
+    STACK_KEY = :__opentelemetry_context_storage__
+    private_constant :KEY, :EMPTY_ENTRIES, :STACK_KEY
```

Comment:
I don't think `KEY` is used anymore?
```suggestion
    EMPTY_ENTRIES = {}.freeze
    STACK_KEY = :__opentelemetry_context_storage__
    private_constant :EMPTY_ENTRIES, :STACK_KEY
```